packages/collections/persistent/_test.pony

```diff
@@ -75,11 +69,11 @@ class iso _TestListApply is UnitTest
     h.assert_eq[U32](l1(0), 1)
     h.assert_eq[U32](l1(1), 2)
     h.assert_eq[U32](l1(2), 3)
-    h.assert_error({()? => l1(3) })
-    h.assert_error({()? => l1(4) })
+    h.assert_error({()? => l1(3)})
+    h.assert_error({()? => l1(4)})
 
     let l2 = Lists[U32].empty()
-    h.assert_error({()? => l2(0) })
```

Comment:
At any rate, if we wanted to change the style here after discussing it, we should probably do it globally for the whole stdlib, and in a separate PR dedicated to that change.